services/codedeploy/src/main/resources/codegen-resources/customization.config

```diff
@@ -5,6 +5,9 @@
     "blacklistedSimpleMethods" : [
         "continueDeployment",
         "skipWaitTimeForInstanceTermination",
-        "updateApplication"
+        "updateApplication",
+        "batchGetApplications",
+        "batchGetDeployments",
```

Comment:
Interesting edge case. I'd imagine it succeeds with none but doesn't return anything obviously. I'd vote for removing any batch get ones.